include/deal.II/lac/petsc_block_sparse_matrix.h

```diff
@@ -174,6 +174,19 @@ namespace PETScWrappers
              const BlockDynamicSparsityPattern &bdsp,
              const MPI_Comm                     com);
 
+      /**
+       * Same as above, but needed in case a BDDC preconditioner must be
+       * initialized.
+       */
+      void
+      reinit(const std::vector<IndexSet>       &local_rows,
+             const std::vector<IndexSet>       &active_rows,
+             const std::vector<IndexSet>       &local_cols,
+             const std::vector<IndexSet>       &active_cols,
+             const BlockDynamicSparsityPattern &bdsp,
+             const MPI_Comm                     com);
```

Comment:
I think the comment is too terse. Can you explain *what* this function does different than the previous one, as opposed to (or in addition to) what it can be used for? You have to be pretty deep into the technical details of BDDC to know what BDDC wants and can't get from the other function :-) (In fact, I have no idea.) You may also want to link to the actual `PreconditionBDDC` class.